www/inc/cdsp.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/**
+ * Wrapper for functionality related to the use of CamillaDSP with moOde
+ */
+
+require_once dirname(__FILE__) . '/playerlib.php';
+
+const CDSP_CHECK_VALID = 1;
+const CDSP_CHECK_INVALID = 0;
+const CDSP_CHECK_NOTFOUND = -1;
+class CamillaDsp {
+
+    private $ALSA_CDSP_CONFIG = '/etc/alsa/conf.d/camilladsp.conf';
+    private $CAMILA_CONFIG_DIR = '/usr/share/camilladsp';
+    private $CAMILA_EXE = '/usr/local/bin/camilladsp';
+    private $device = NULL;
+    // private $mode = 'cdsp';
+    private $configfile = NULL;
+
+    function __construct ($configfile, $device = NULL) {
+        $this->configfile =$configfile;
+        $this->device = $device;
+    }
+
+    /**
+     * Set in camilladsp config file the playback device to use
+     */
+    function setPlaybackDevice($device) {
+        if( $this->configfile != NULL && $this->configfile != 'off' && $this->configfile != 'custom') {
+            $this->device = $device;
+            sysCmd("sudo sed -i -s '/device/s/hw:[0-9]/hw:" . $device . "/g' " . $this->getCurrentConfigFileName() );
+        }
+    }
+
+    /**
+     * Set in the alsa_cdsp config the camilladsp config file to use
+     */
+    function selectConfig($configname) {
+        if($configName != 'custom' && $configName != 'off') {
+            $configfilename = $this->CAMILA_CONFIG_DIR . '/configs/' . str_replace ('/', '\/', $configname);
+            $configfilename = str_replace ('/', '\/', $configfilename);
+            syscmd("sudo sed -i -s '/[ ]config_out/s/\\\".*\\\"/\\\"" . $configfilename . "\\\"/g' " . $this->ALSA_CDSP_CONFIG );
```

Comment:
To be honest the current PR is far from what I did want to deliver, the intention was to also include:
* The camilladsp webgui (just as experiment, integrationwith alsa_cdsp isn't optimal)
* Easy single convolution with a gain, just for basic room/headphone correction
* Equalizer APO config converter
* See if it is possible to integrate ASH IR Dataset, with simple selectors

When your PR was merged, I also had already my own version of changing the output device. I assumed that I had some more time to finish my stuff :-) Before things got more out of sync, decided to merge the part that was already finished.

I considered the symbolic link, but because I don't know how the this combines with the CamillaDSP socket api/webgui (which also includes configuration management), I decided to leave it out until I reach that point.